perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py

```diff
@@ -198,18 +278,18 @@ def _execute_pkb_cmd(pkb_cmd, stdout_filename, stderr_filename):
   # Will probably have to implement with threading.
   p.wait()
   elapsed_time = time.time() - start_time
+  if elapsed_time == 1:
```

Comment:
From running this benchmark and having it fail many many times :). When something went wrong the phase duration was almost always 1s (save 512 thread count error). The phase duration will never be that short and therefore is a good indicator when the call fails. I was considering including the traceback from stderr in a later version.

Do you disagree with this logic?